### PR TITLE
Converted github md links to relative links to html

### PR DESCRIPTION
**Pull request by bgrant0607**
cc @thockin @RichieEscarez @caesarxuchao

### CODE REVIEW

File: gettingstarted.md [hunk 1]
```diff
@@ -33,31 +33,31 @@ hostedservices:
...
-    url: https://github.com/GoogleCloudPlatform/kubernetes/blob/master/docs/getting-started-guides/aws.md
+    url: v1.0/docs/getting-started-guides/aws.html
   - label: Mesos
-    url: https://github.com/GoogleCloudPlatform/kubernetes/blob/master/docs/getting-started-guides/mesos.md
+    url: v1.0/docs/getting-started-guides/mesos.html
 
 firstapp:
     label: Run Now
-    url: https://github.com/GoogleCloudPlatform/kubernetes/blob/master/examples/guestbook/README.md
+    url: examples/guestbook/README.html
```
Comment (caesarxuchao) [inline gettingstarted.md]: add v1.0?

**Comment by thockin** [comment]
Once we commit this, we're basically live right?

**Comment by thockin** [comment]
LGTM, but let's coordinate commits

File: gettingstarted.md (hunk 1 above)
Comment (bgrant0607) [inline gettingstarted.md]: Done

**Comment by bgrant0607** [comment]
Merging to make progress.

**Pull request merged**